include/GafferImage/ImageReader.h

```diff
@@ -110,10 +110,18 @@ class ImageReader : public ImageNode
 		Gaffer::StringPlug *colorSpacePlug();
 		const Gaffer::StringPlug *colorSpacePlug() const;
 
+		Gaffer::StringPlug *colorSpaceMetadataNamePlug();
```

Comment:
alright, I think that would work with passing the full image metadata.

Could you provide an example to add the content of CompoundObjectPlug to the CompoundData, I'm assuming that I would use the ImagePlug->metadataPlug() as a source?